### PR TITLE
fix(issues): Reduce shift when switching replays

### PR DESCRIPTION
**Pull request by scttcper**
Reduces layout shift when switching between replays on the issue replays tab.

Adds a placeholder for some text that appears after load
image

### CODE REVIEW

File: static/app/views/issueDetails/groupReplays/groupReplays.tsx [hunk 1]
```diff
@@ -242,11 +252,11 @@ function GroupReplaysTable({
...
       <ReplayCountHeader>
-        <StyledIconUser size="sm" />
+        <IconUser size="sm" />
         {t(
-          'Replay captured %s users experiencing this issue across %s events.',
-          replayCount,
-          group.count
+          'Replay captured %s experiencing this issue across %s.',
+          tn('%s user', '%s users', replayCount ?? 0),
+          tn('%s event', '%s events', group.count)
```
Comment (ryan953) [inline static/app/views/issueDetails/groupReplays/groupReplays.tsx:259]: Could we put the whole sentence in to the ternary and tn() calls. drop the t() wrapper part. Should be easier for  translators to have the full sentence there.

Comment (scttcper) [reply static/app/views/issueDetails/groupReplays/groupReplays.tsx:259]: i was thinking you could have 1 user but 2 events or 2 events and 1 user

**Review by ryan953 — APPROVED** [review]
Rest looks good. Data-test-id all the things!

File: static/app/views/issueDetails/groupReplays/groupReplays.tsx (hunk 1 above)
Comment (ryan953) [reply static/app/views/issueDetails/groupReplays/groupReplays.tsx:259]: Fair enough. 

Tough to use t18n lib :(

**Pull request merged**